### PR TITLE
LiveLogger: Don't blink

### PR DESCRIPTION
**Pull request by rainersigwald**
I keep finding the blinking distracting and would like to propose removing it.

### CODE REVIEW

File: src/MSBuild/LiveLogger/ANSIBuilder.cs [hunk 1]
```diff
@@ -196,11 +196,6 @@ public static string DoubleUnderlined(string text)
                 return String.Format("\x1b[21m{0}\x1b[24m", text);
             }
 
-            public static string Blinking(string text)
```
Comment (marcpopMSFT) [inline src/MSBuild/LiveLogger/ANSIBuilder.cs:199]: Will we ever want blink anywhere else?

Comment (rainersigwald) [reply src/MSBuild/LiveLogger/ANSIBuilder.cs:199]: I don't think I personally would . . .

Comment (edvilme) [reply src/MSBuild/LiveLogger/ANSIBuilder.cs:199]: I can only imagine it in cases where user attention is needed. But as this is a logger (and not interactive) I can't think of any relevant scenario

**Review by Forgind — APPROVED** [review]
~Can we change the api base to 17.6.0-preview-something?~

See edvilme's comment

**Comment by edvilme** [comment]
I think line 62 of ProjectNode.cs got overwritten at merge, as it still references `ANSIBuilder.Formatting.Blinking`.
https://github.com/dotnet/msbuild/pull/8406/files#file-src-msbuild-livelogger-projectnode-cs-L62

**Review by JanKrivanek — APPROVED** [review]
Looks good (once the merge reintroduced usage is again removed)

**Comment by rainersigwald** [comment]
Sorry for the delay, merge un-broken now.

**Pull request merged**